e2e/testdata/hydration/compiled-json/helm-components/wordpress/secret_my-wordpress.json

```diff
@@ -0,0 +1,28 @@
+{
+	"apiVersion": "v1",
+	"items": [
+		{
+			"apiVersion": "v1",
+			"data": {
+				"wordpress-password": "YWJjZGVmZw=="
+			},
+			"kind": "Secret",
+			"metadata": {
+				"annotations": {
+					"config.kubernetes.io/origin": "configuredIn: kustomization.yaml\nconfiguredBy:\n  apiVersion: builtin\n  kind: HelmChartInflationGenerator\n"
+				},
+				"labels": {
+					"app.kubernetes.io/instance": "my-wordpress",
+					"app.kubernetes.io/managed-by": "Helm",
+					"app.kubernetes.io/name": "wordpress",
+					"helm.sh/chart": "wordpress-15.2.35",
+					"test-case": "hydration"
+				},
+				"name": "my-wordpress",
+				"namespace": "wordpress"
+			},
+			"type": "Opaque"
+		}
+	],
+	"kind": "List"
+}
```

Comment:
nit: linebreak

Reply:
This is one of many autogenerated JSON files. The test fails if you change the checksum of the file.